lib/bloc/payment_options/payment_options_bloc.dart

```diff
@@ -29,22 +29,26 @@ class PaymentOptionsBloc extends Cubit<PaymentOptionsState> {
     emit(state.copyWith(channelFeeLimitMsat: channelFeeLimitMsat));
   }
 
-  Future<void> resetPaymentOptions() async {
-    try {
-      await _preferences.setPaymentOptionsProportionalFee(kDefaultProportionalFee);
-      await _preferences.setPaymentOptionsExemptFee(kDefaultExemptFeeMsat);
-      await _preferences.setPaymentOptionsChannelSetupFeeLimit(kDefaultChannelSetupFeeLimitMsat);
-      emit(const PaymentOptionsState.initial());
-      _log.info(
-        "Reverted payment options to default values: "
-        "proportionalFee: $kDefaultProportionalFee "
-        "exemptFeeMsat: $kDefaultExemptFeeMsat "
-        "channelFeeLimitMsat: $kDefaultChannelSetupFeeLimitMsat",
-      );
-    } catch (e) {
-      _log.severe("Failed to reset payment options.");
-      rethrow;
+  Future<void> resetPaymentOptions([bool checkPaymentOptions = false]) async {
```

Comment:
`resetPaymentOptions` can be renamed to `setDefaultPaymentOptions` as we're not really resetting anything when shared preferences does not have payment options.